### ISSUE TITLE
Gamer Arena

### ISSUE THREAD

**Issue by olowudaniel1** [thenewboston-developers/Projects#249]
# Project Details

#### How would you describe your project?
This project simply entails organizing online game tournaments where the winners (1st, 2nd & third) gets TNBC.  Examples of these games are CHESS, FIFA, F1, More games would be decided by the community or team members.

#### What problem does this project solve?
This project creates an  opportunity for people to earn the coins especially people that are not developers.

#### Who is the target market?
Everyone that can play the games available.

#### How will this project benefit the network and/or the community?
This project creates more awareness for TNBC and would drastically increase the community members even while  in alpha stage. There are a lot of people that are interested in getting the coins but they are not developers. Also people are going to refer their friends to play these games so they stand a chance to earn the coin. This is also a good marketing tactics to get more people

#### How is this project centered around thenewboston?
TNBC is the only coin that would be used for this

#### What is the estimated completion date for this project? (rough estimates are acceptable)
A month or two. 

#### Please include a detailed roadmap for your project below. This must include a list of all milestones where payments will be awarded upon completion of each.
1. Wireframing, sitemap and user flow
2. Hi-fi Design
3. Web Design and Development
 
More to be decided as project progresses.

#### What is your Discord username?
Weird Guy#6773

# Milestone Details

Please include a detailed breakdown of the first milestone including a list of all quantifiable objectives. Upon 
completion of the milestone, each objective will be reviewed by thenewboston team and once all objectives have been 
verified as completed, coins will then be rewarded.

Note that all future detailed milestone breakdowns will be written as a separate GitHub issue. Only the first milestone 
is included in this project proposal.

---

#### First Milestone Description
Wireframe & Userflow  Design

#### First Milestone Objectives

Creating the wireframe, sitemap and user flow. The figma file would be presented as prrof of work.

#### How long do you estimate this first milestone will take to complete?
2 weeks max

#### Project Wallet
86ee75d4cea77ca8339cc97f5e7426a3434d9f91465cd441c0c7593bd852e955

**Comment by tspearing** [comment]
Hi @olowudaniel1, let's pretend this is approved for X coins. What is the plan for the flow of the TNBC?

How many tournaments? 
How much do 1st, 2nd and 3rd win?
What % of X will be paid out? Any held back?

Please include wireframe and sitemap as a part of the proposal. It sounds like this is a website? Please describe in more detail.

**Comment by olowudaniel1** [comment]
The number of tournaments would be decided by team member of the project. it could be a tournament per month or a tournament  in three months  for each games or we could run two tournaments of each games synchronously.

For the flow of TNBC, the users would register for the tournaments and the fee would be so  little, could be 1,000 TNBC or less. The winners are going to get  remarkable returns. For example, if they register for a tournament with 1,000 TNBC. 1st wins 50,000 TNBC, 2nd, 30,000 TNBC, 3rd, 20,000  TNBC. These are high returns compared to the registration fee. 

Yes, there has to be held back. This would be used for the sustenance of the project, marketing  and to pay team members.

Yes, this would be a web app and we would also have a discord channel for communication. As the project progresses, we could end up having a mobile app. The web app, would contain the landing page and would provide information such as the leaderboard, tournaments, what TNBC is about and other relevant information. The users would also have a dashboard. This dashboard would enable them do quite a number of things but the highest priority is the wallet. Where their funds would be kept and where they would paid.

**Comment by manishram** [comment]
Hey @olowudaniel1 Chrisha from Marketing Team will reach you out on this one.

**Comment by olowudaniel1** [comment]
Thanks for the response. I would await her message

**Comment by Chrisha1212** [comment]
> Thanks for the response. I would await her message

Hey @olowudaniel1 do you have discord account? if yes can you reach out to me MsChrishaph#0709 so we can start discussing about the plan we need .

**Comment by olowudaniel1** [comment]
Hello

**Comment by hussu010** [comment]
@olowudaniel1, I suggest taking the fees (minimum) in TNBC for participation in the tournaments since that would encourage people to acquire tnbc through contributions, faucets, and rains.

**Comment by olowudaniel1** [comment]
@hussu010  Yeah. The marketing team would decide how much would be required for registration.. It could be as low as 50TNBC

**Comment by manishram** [comment]
> Hello
Hey @olowudaniel1 you wanted to say something?

**Comment by tspearing** [comment]
Hi @olowudaniel1, does [this](https://www.figma.com/file/dutfNRhzRTKzQ1zcxYsOPr/tournament?node-id=0%3A1) represent the design for the blog of the first milestone? Bojan tells us that some blog designs were done.

**Comment by olowudaniel1** [comment]
@tspearing  Yes please

**Comment by olowudaniel1** [comment]
@manishram  I was told the project is no longer under marketing and that the audit team would re-open it
